### PR TITLE
Vandana

### PR DESCRIPTION
**Pull request by varalakshmitheegala**
Please review and suggest if there are any changes

### CODE REVIEW

File: debitFeature [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This is debit card
+adding this feature
```
Comment (aravindpraneeth) [inline debitFeature:2]: WTF.. Can you please add another line after 2 and line.

**Comment by aravindpraneeth** [comment]
Can you please check the image again

File: debitFeature (hunk 1 above)
Comment (varalakshmitheegala) [reply debitFeature:2]: resolved

**Review by aravindpraneeth — APPROVED** [review]
Looks good to me. Approving it.

**Pull request merged**